README.md

```diff
@@ -1324,6 +1324,22 @@
     };
     ```
 
+  - Self invoking constructors to facilitate invoking constructors with or without ``new`` keyword.
+
+    ```javascript
+    function Jedi(options) {
+      options || (options = {});
+      if (!(this instanceof Jedi)) {
```

Comment:
Thanks @goatslacker, was not aware of the [Cannot call a class as a function](https://babeljs.io/repl/#?experimental=true&playground=true&evaluate=true&loose=false&spec=false&code=class%20Employee%20%7B%0A%20%0A%20%20constructor(name)%20%7B%0A%20%20%20%20this.name%20%3D%20name%3B%0A%20%20%7D%0A%0A%20%20get%20name()%20%7B%0A%20%20%20%20return%20this._name.toUpperCase()%3B%0A%20%20%7D%0A%0A%20%20set%20name(name)%7B%0A%20%20%20%20if(name)%7B%20%0A%20%20%20%20%20%20this.name%20%3D%20name%3B%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A%0Avar%20employee%20%3D%20Employee('John')%3B%0A) error.